### DISCUSSION TITLE
spam

### DISCUSSION THREAD

**Discussion by jintor** [cdnjs/cdnjs#14111] [General] [answered]
Hi,

I emphasize on WHILE because when I search about it I find stuff about recording a session (canvas) then (ONLY when the user hit stop, it can be sent to the host server.....)

it's ok for small sessions (1 min, 2 min), but what about 1 hour, 2 hours, etc ????

I would like : that when the user begins the session (hit start), then it starts to live send chunks **while** the session is still going on.

is it possible ?

is there a javascript library that could give me a head start ?

**Comment by MattIPv4** [comment, marked as answer]
👋 I'm not really sure what you're asking for. This GitHub repo is meant for support with cdnjs itself.

I'd recommend posting on a site such as [DigitalOcean's Community](https://www.digitalocean.com/community) or [StackOverflow](https://stackoverflow.com/).
